skipped_tests.toml

```diff
@@ -7,83 +7,114 @@ unit = [
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_absent_when_stage_is_invalid",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_absent_when_stage_is_valid_and_only_one_stage_is_associated_to_deployment",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_absent_when_stage_is_valid_and_two_stages_are_associated_to_deployment",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_associating_to_new_deployment_errored_on_api_creation",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_associating_to_new_deployment_errored_on_model_creation",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_associating_to_new_deployment_errored_on_resource_creation",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_exists_and_is_to_associate_to_existing_deployment",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_is_already_at_desired_deployment",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_stage_is_already_at_desired_deployment_and_needs_stage_variables_update",
+	" tests/unit/states/test_boto_apigateway.py::BotoApiGatewayTestCase::test_present_when_swagger_file_is_invalid",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_absent_when_a_single_pool_exists",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_absent_when_erroring_on_delete_identity_pool",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_absent_when_failing_to_describe_identity_pools",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_absent_when_multiple_pool_exists_and_removeallmatched_flag_is_true",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_absent_when_pool_does_not_exist",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_absent_when_removeallmatched_is_false_and_multiple_pools_matched",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_present_when_failing_to_create_a_new_identity_pool",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_present_when_failing_to_describe_identity_pools",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_present_when_failing_to_get_identity_pool_roles",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_present_when_failing_to_update_an_existing_identity_pool",
+	" tests/unit/states/test_boto_cognitoidentity.py::BotoCognitoIdentityTestCase::test_present_when_multiple_pools_with_same_name_exist",
```

Comment:
are the spaces in these items intentional and have any special meaning? I'm not really familiar with it and maybe it's obvious.

Reply:
No, that was me messing up my sam commands I used to edit the text I copy pasted from `pytest` output.